### PR TITLE
Add test for AutoUpdateProfile.domains

### PR DESCRIPTION
**Pull request by chrismedrela**
Just to prevent situation like the one described in first point in this comment: https://github.com/swcarpentry/amy/pull/825#issuecomment-229015225

### CODE REVIEW

File: workshops/test/test_autoupdate_profile.py [hunk 1]
```diff
@@ -36,7 +43,7 @@ def test_update_profile(self):
             'url': '',
             'username': 'changed',
             'affiliation': '',
-            'domains': [],
+            'domains': [self.chemistry.pk],
             'lessons': [self.python.pk, self.matlab.pk],
```
Comment (pbanaszkiewicz) [inline workshops/test/test_autoupdate_profile.py]: Please extend to languages as well.

File: workshops/test/test_autoupdate_profile.py [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
         self._setUpAirports()
+
         self.user = Person.objects.create_user(
             username='user', personal='', family='',
             email='user@example.org', password='pass')
+
         self.python = Lesson.objects.create(name='swc/python')
         self.shell = Lesson.objects.create(name='swc/shell')
         self.matlab = Lesson.objects.create(name='swc/matlab')
         self.git = Lesson.objects.create(name='swc/r')
```
Comment (pbanaszkiewicz) [inline workshops/test/test_autoupdate_profile.py]: There's `self._setUpLessons()` avaliable (defined in `base.py`), which you can perhaps extend to add two other lessons.

File: workshops/test/test_autoupdate_profile.py (hunk 1 above)
Comment (chrismedrela) [inline workshops/test/test_autoupdate_profile.py]: That was tricky.

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, ping

**Comment by pbanaszkiewicz** [comment]
Thanks @chrismedrela!

**Pull request merged**